vaadin-combo-box-flow-parent/vaadin-combo-box-flow-demo/pom.xml

```diff
@@ -22,6 +22,14 @@
       <groupId>com.vaadin</groupId>
       <artifactId>flow-html-components</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.vaadin</groupId>
+      <artifactId>flow-lit-template</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.vaadin</groupId>
+      <artifactId>flow-polymer-template</artifactId>
+    </dependency>
```

Comment:
Removed. It was added into original repo, because otherwise the demo application refused to render the UI. But it works in monorepo without these deps, so could be removed.